source/conf.py

```diff
@@ -209,7 +209,7 @@
 # If true, "Created using Sphinx" is shown in the HTML footer. Default is True.
 #html_show_sphinx = True
 
-# If true, "(C) Copyright ..." is shown in the HTML footer. Default is True.
+# If true, "(C) Copyripght ..." is shown in the HTML footer. Default is True.
```

Comment:
Typo?

Reply:
It is a comment which will not appear anywhere, you can safely ignore it